pkg/admission/util.go

```diff
@@ -75,3 +76,18 @@ func convert2Namespace(obj interface{}) *v1.Namespace {
 	log.Logger().Warn("cannot convert to *v1.Namespace", zap.Stringer("type", reflect.TypeOf(obj)))
 	return nil
 }
+
+// generate appID based on the namespace value,
+// and the max length of the ID is 63 chars.
```

Comment:
nit: update comment to align with implementation

Reply:
updated.